test/stub/stub.go

```diff
@@ -1,33 +1,35 @@
 package stub
 
 import (
-	buildv1alpha1 "github.com/shipwright-io/build/pkg/apis/build/v1alpha1"
+	buildv1beta1 "github.com/shipwright-io/build/pkg/apis/build/v1beta1"
 
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 // BuildRunEmpty simple empty BuildRun instance.
-func BuildRunEmpty() buildv1alpha1.BuildRun {
-	return buildv1alpha1.BuildRun{}
+func BuildRunEmpty() buildv1beta1.BuildRun {
+	return buildv1beta1.BuildRun{}
 }
 
 // TestBuild returns instance of Build for testing purposes
-func TestBuild(name, image, source string) *buildv1alpha1.Build {
-	strategyKind := buildv1alpha1.ClusterBuildStrategyKind
+func TestBuild(name, image, source string) *buildv1beta1.Build {
+	strategyKind := buildv1beta1.ClusterBuildStrategyKind
 
-	result := &buildv1alpha1.Build{
+	result := &buildv1beta1.Build{
 		ObjectMeta: metav1.ObjectMeta{
 			Name: name,
 		},
-		Spec: buildv1alpha1.BuildSpec{
-			Strategy: buildv1alpha1.Strategy{
+		Spec: buildv1beta1.BuildSpec{
+			Strategy: buildv1beta1.Strategy{
 				Name: "buildah",
 				Kind: &strategyKind,
 			},
-			Source: buildv1alpha1.Source{
-				URL: &source,
+			Source: &buildv1beta1.Source{
```

Comment:
This is not valid without source.type being set.

Reply:
i have set the type to `Git`